### PR TITLE
Add Item Class and Tests

### PR DESCRIPTION
**Pull request by aschwartz1**
As per the Iteration 0 Spec at the [end of this page.](https://backend.turing.io/module1/projects/black_thursday/iteration_0)

### CODE REVIEW

File: test/item_test.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+require 'bigdecimal'
+require './lib/item'
+
+class ItemTest < Minitest::Test
+  def setup
+    @pencil = Item.new({
+      id:           1,
+      name:         'Pencil',
+      description:  'You can use it to write things.',
+      unit_price:   BigDecimal.new(10.99, 4),
```
Comment (aschwartz1) [inline test/item_test.rb]: @jbranham1 Should we look into this?

File: test/item_test.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'bigdecimal'
+require './lib/item'
+
+class ItemTest < Minitest::Test
+  def setup
+    @pencil = Item.new({
+      id:           1,
```
Comment (aschwartz1) [inline test/item_test.rb]: I've no idea about this style suggestion...

File: test/item_test.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+require 'minitest/autorun'
```
[the remark this answers is not in the document]
Comment (jbranham1) [reply test/item_test.rb:1]: @aschwartz1 Wonder why this is still showing?

File: test/item_test.rb (hunk 2 above)
Comment (jbranham1) [reply test/item_test.rb]: @aschwartz1 Does it need to be id:  1?

File: test/item_test.rb (hunk 1 above)
Comment (jbranham1) [reply test/item_test.rb]: Yes! We probably need to do what hound suggests! @aschwartz1

**Review by jbranham1 — APPROVED** [review]
Went through this together this morning and worked through Hound reviews. Looks great!

**Pull request merged**